<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogArx</class>
 <widget class="QDialog" name="DialogArx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>525</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Parametry ARX</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>1101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>601</width>
     <height>291</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>599</width>
      <height>289</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="dodaj_wspolczynnik">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>103</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd"/>
   </property>
  </widget>
  <widget class="QPushButton" name="usun_wspolczynnik">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>103</width>
     <height>29</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListRemove"/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Opóżnienie transportowe (k)</string>
   </property>
  </widget>
  <widget class="QSlider" name="opoznienie_suwak">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="opoznienie_wartosc">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>20</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Ograniczenia na wyjściu</string>
   </property>
   <widget class="QDoubleSpinBox" name="ograniczenie_wyj_max">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ograniczenie_wyj_min">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>0.000000000000000</double>
    </property>
    <property name="value">
     <double>-10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Maksimum:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Ograniczenia na wejściu</string>
   </property>
   <widget class="QDoubleSpinBox" name="ograniczenie_wej_max">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ograniczenie_wej_min">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>0.000000000000000</double>
    </property>
    <property name="value">
     <double>-10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Maksimum:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>180</y>
     <width>631</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Zakłócenia</string>
   </property>
   <widget class="QDoubleSpinBox" name="zaklocenie_wartosc">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="zaklocenie_suwak">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="pageStep">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Odchylenie standardowe</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>29</y>
      <width>611</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="chart_zaklocenia_view"/>
   </widget>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Współczynniki</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogArx</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogArx</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
